packages/cloud_firestore/cloud_firestore_web/lib/src/interop/firestore.dart

```diff
@@ -477,7 +477,7 @@ class Query<T extends firestore_interop.QueryJsImpl>
         ? [snapshot.jsObject]
         : fieldValues!.map(jsify).toList();
 
-    return callMethod(method, 'apply', jsify([null, args]));
+    return callMethod(method, 'apply', [null, jsify(args)]);
```

Comment:
Not sure this change is correct 🤔 how come this was changed

Reply:
I try with the old way, it also works, so I'm reverting this :)

Reply:
See internal chat - fixes an issue on prod

Reply:
Nvm, have seen the internal chat for this